client/lib/domains/resolve-domain-status.js

```diff
@@ -150,6 +149,14 @@ export function resolveDomainStatus(
 				};
 			}
 
+			if ( isDomainOnlySite ) {
+				return {
+					statusText: translate( 'Parked' ),
+					statusClass: 'status-parked',
```

Comment:
I think we can change this to `status-pending` - I'm working on the colors in a separate PR - https://github.com/Automattic/wp-calypso/pull/44335 and that'll use the proper gray. Though you can hold on till we get feedback from @fditrapani in there first

Reply:
Let's remove the `status-parked` in another PR, perhaps after your changes have been merged.